pcdsdevices/gauge.py

```diff
@@ -204,7 +204,7 @@ class GCCPLC(GaugePLC):
     high_voltage_disable = Cpt(EpicsSignalRO, ':HV_DIS_DO_RBV', kind='normal',
                                doc=('enables the high voltage on the cold '
                                     'cathode gauge'))
-    protection_setpoint = Cpt(EpicsSignalRO, ':PRO_SP_RBV', kind='normal',
+    protection_setpoint = Cpt(EpicsSignalWithRBV, ':PRO_SP', kind='normal',
```

Comment:
In the future, we should use `PytmcSignal` for all of these instead of bouncing between using `EpicsSignalRO` and providing the `RBV` or using `EpicsSignalWithRBV` and omitting it.